common/src/main/java/com/redhat/lightblue/rest/cors/EbayCorsFilterRegistration.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.lightblue.rest.cors;
+
+import org.ebaysf.web.cors.CORSFilter;
+
+import javax.servlet.FilterRegistration;
+import javax.servlet.ServletContext;
+import java.util.Iterator;
+import java.util.List;
+
+public class EbayCorsFilterRegistration implements CorsFilterRegistration {
```

Comment:
They have a simple CORS filter already implemented. If we ever adopt KeyCloak we can use that instead.

https://github.com/ebay/cors-filter